ibft/ibft_network.go

```diff
@@ -0,0 +1,32 @@
+package ibft
+
+import "github.com/bloxapp/ssv/network"
+
+func (i *ibftImpl) listenToNetworkMessages() {
+	msgChan := i.network.ReceivedMsgChan()
+	go func() {
+		for msg := range msgChan {
+			i.msgQueue.AddMessage(&network.Message{
+				Lambda:        msg.Message.Lambda,
+				SignedMessage: msg,
+				Type:          network.NetworkMsg_IBFTType,
```

Comment:
GetTopic is depend on the msg validatorPublicKey so we need to add it to all msg's

Reply:
not sure what you mean:?

Reply:
we need to pass validatorPublicKey for the GetTopic purpose

Reply:
you have that inside the signed msg